app/src/main/java/org/xbmc/kore/jsonrpc/type/AudioType.java

```diff
@@ -279,6 +279,7 @@ public DetailsAlbum(JsonNode node) {
         public static final String ALBUMARTIST = "albumartist";
         public static final String ALBUMARTISTID = "albumartistid";
         public static final String ALBUMID = "albumid";
+        public static final String ARTISTID = "artistid";
```

Comment:
This is not needed. `DetailsAlbum` inherits from `DetailsMedia` which already has `artistid` (populated).